src/pod.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * This file is part of cc-oci-runtime.
+ *
+ * Copyright (C) 2016 Intel Corporation
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+#ifndef _CC_POD_H
+#define _CC_POD_H
+
+#include <stdbool.h>
+
+#include <glib.h>
+
+#include "util.h"
+#include "oci.h"
+
+#define CC_POD_OCID_ANNOTATION_NAMESPACE "ocid/"
+#define CC_POD_OCID_ANNOTATION_NAMESPACE_SIZE 5
```

Comment:
Do we really need this? It's only referenced once and we could just use `g_strcmp0()` since `CC_POD_OCID_ANNOTATION_NAMESPACE` is bounded anyhow?